common/utils.js

```diff
@@ -34,7 +34,57 @@ export async function getBufferFromUrl(url) {
   return arrayBuffer;
 }
 
-export async function buildConstantByNpy(builder, url) {
+// ref: http://stackoverflow.com/questions/32633585/how-do-you-convert-to-half-floats-in-javascript
+export const toHalf = (function() {
```

Comment:
This method has been added in @mingmingtasd's PR https://github.com/webmachinelearning/webnn-samples/pull/226. Please help review and merge that one then we can ignore this change in your PR.

Reply:
SGTM, I'll rebase once #226 merged.

Reply:
#226 may not need `toHalf` helper once it uses cast operator.